topostats/grains.py

```diff
@@ -106,6 +109,16 @@ def gaussian_filter(self, **kwargs) -> np.array:
             mode=self.gaussian_mode,
             **kwargs,
         )
+    
+    def z_thresholding(self, **kwargs) -> np.array:
+        """Apply height thresholding so outliers are capped at config values"""
```

Comment:
Pedantic but "capped" usually means or is interpreted as an upper limit, but outliers are also removed at the lower limit. Perhaps "clipped" would be more appropriate.

Reply:
Yep - we aim to make it as easy to read as possible!